pkg/distro/defs/fedora/distro.yaml

```diff
@@ -822,26 +827,39 @@ image_types:
             - "xfsprogs"
             - "xz"
             - "zram-generator"
-          condition:
-            version_less_than:
-              "41":
+          conditions:
+            "f40 and below use dnsmasq":
+              when:
+                version_less_than: "41"
+              append:
                 include:
                   - "dnsmasq"
-              "42":
+            "f41 and below uses parsec/zezere":
+              when:
+                version_less_than: "42"
+              append:
                 include:
                   - "dbus-parsec"
                   - "kernel-tools"
                   - "parsec"
                   - "policycoreutils-python-utils"
                   - "zezere-ignition"
-              "43":
+            "f42 and below uses basesystem":
+              when:
+                version_less_than: "43"
+              append:
                 include:
                   - "basesystem"
-            version_greater_or_equal:
-              "41":
+            "f41+ needs bootupd":
```

Comment:
Maybe consistently say above and below instead of using below and +?
eg. "f41 and above need bootupd"